android-agent/src/test/java/io/opentelemetry/android/OpenTelemetryRumBuilderTest.java

```diff
@@ -348,4 +344,8 @@ private static ServiceManager createServiceManager(Object... services) {
     private OpenTelemetryRumBuilder makeBuilder() {
         return OpenTelemetryRum.builder(application, buildConfig());
     }
+
+    private OtelRumConfig buildConfig() {
```

Comment:
Since this is called in several tests and doesn't simply create a `OtelRumConfig` instance that just uses the defaults, should the method name indicate that?